core/src/main/java/feast/core/model/FeatureTable.java

```diff
@@ -147,6 +155,13 @@ public static FeatureTable fromProto(
       table.setStreamSource(DataSource.fromProto(spec.getStreamSource()));
     }
 
+    table.setMetadataHash(
+        Objects.hash(
```

Comment:
won't it be more stable to use hash of proto?
of course we would need to presort all lists